### DISCUSSION TITLE
Is it normal behaviour for data to be wiped out on paused apps?

### DISCUSSION THREAD

**Discussion by bradpotts** [supabase/supabase#14054] [Questions]
My app went on pause when I un-paused it all my data in storage and database information was wiped out and removed. Just wondering if this is normal and expected or is there an error?

**Comment by GaryAustin1** [comment]
That is not expected.  All data and storage should be there after pause/restore.

**Comment by silentworks** [comment]
Hey @bradpotts, I think this is being handled by the support team at the moment. Are you OK with closing this one out?

**Comment by UDsGitHub** [comment]
I just experienced this same issue. I dont know if this has been completely resolved @silentworks

**Reply by GaryAustin1** [reply]
You should either start a new discussion with more info, or file a support request if your data is gone.
This is 2 year old and not at all clear was a SB issue.

**Reply by UDsGitHub** [reply]
I did a support request on supabase to see if they could help me restore my data

**Reply by UDsGitHub** [reply]
I also dont have more info to add, because I didnt take screenshots of the table before the pause. My issue is essentially the same problem as OP

**Reply by UDsGitHub** [reply]
Ignore my post, somehow the tables and data returned as I checked just now
